vpype_viewer/qtviewer/viewer.py

```diff
@@ -443,6 +445,8 @@ def show(
     view_mode: ViewMode = ViewMode.PREVIEW,
     show_pen_up: bool = False,
     show_points: bool = False,
+    pen_width: bool = 0.3,
+    pen_opacity: bool = 0.8,
```

Comment:
The corresponding changes must be made in the docstring.

Reply:
hi. where should i do changing the doc's content?

Reply:
In this function's docstring.

Reply:
check :-) thanks for remembering